exampleScreenCapturer/src/main/java/com/twilio/video/examples/screencapturer/ScreenCapturerActivity.java

```diff
@@ -49,6 +50,18 @@ public void onCreate(Bundle savedInstanceState) {
         localVideoView = (VideoView) findViewById(R.id.local_video);
     }
 
+    @Override
```

Comment:
Can you scope the screen capturer manager to onCreate/onDestroy?

This way the user can continue capturing the screen in the background with a notification visible.

Reply:
I think you can remove the onStart onStop if they are not be used